rukundoeric/calculator
# Milestone 4: Logic

## What does this do? 
*  Calculator
## Description of Task to be completed? 
*  In this milestone I implemented the logic part of the calculator.
## How should this be manually tested? 
NOTE: Make sure you have Node.js and npm or yarn installed on your PC.
- Clone this repo: `git clone https://github.com/rukundoeric/calculator.git`
- Navigate into the project directory: `cd calculator`
- Install all packages: `yarn install or npm install`
- Start the app: `yarn start or npm start`
- or just open Live-Demo [HERE](https://calculator-2v.herokuapp.com/)
